hal/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "atsamx7x-hal"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
`boards/` crates (and lock files) should also be bumped.

Reply:
Technically, no difference as we haven't added any code change. It's formal release to get documentation generated on `docs.rs`.
Updated.